lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -318,6 +318,9 @@ public function createDatabase($database = null)
      */
     public function dropAutoincrement($table)
     {
+        if ($table instanceof Table) {
```

Comment:
as an alternative we can fix this on the callers end:

https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Schema/OracleSchemaManager.php#L334

Your choice ...